app/models/bracket.rb

```diff
@@ -12,5 +12,36 @@ class Bracket < ApplicationRecord
   has_many :entries, dependent: :destroy
   has_many :matchups, dependent: :destroy
 
+# NEED TO MAKE AN after_create ACTION TO AUTOMATICALLY RUN THE METHODS WHEN A BRACKET IS CREATED (like in periods model of status_reporting)
+
+  # make a validation that the number of entries equals an acceptable bracket (8, for now; add 4 and 16 later; add power of 2 later), if not then the bracket needs to fail to create
+
   enum status: { created: "created", in_progress: "in_progress", completed: "completed" }
+  # enum number_of_entries: { eight: 8 } # gonna leave this in to come back to later on, might not need it/reformat it
+
+  def create_entries
+    self.number_of_entries.times do |i|
+      Entry.create!(bracket: self, seed: i + 1)
```

Comment:
You can leverage your association here

```suggestion
      bracket.entries.create!(seed: i + 1)
```